map.js

```diff
@@ -159,7 +180,7 @@ function drawMapOverlay(){
             hloc.draw(ctx, mouseLocInMapCoords);
         }
     }
-    else if(currentOverlay == "Exploration"){
+    else if(currentOverlay == "Exploration"){//We may not really need this third overlay, since we can just draw the TSP underneath the normal icons.
```

Comment:
I can't think of another reason to have a third overlay at the moment.

we have locations and nirnroot.
and they each have TSPs that can be drawn on a toggle.

is there a 3rd set/kind of map data that we want to display?

Reply:
I think locations and nirnroot is it for now.

Reply:
Okay, I dropped a TODO/reminder to remove it after we redo UI.